### DISCUSSION TITLE
Popover actions with Dom nodes (Non-Anvil components)

### DISCUSSION THREAD

**Discussion by chad63e** [anvilistas/anvil-extras#318] [Q&A] [answered]
I have previously inquired about this on the Anvil forum (https://anvil.works/forum/t/non-anvil-component-popovers/11929), but I am still not sure how to get this working.  

I have an SVG element within a Link that I would like to attach a popover.  This works fine if the trigger method is "hover" or something automatic.  However, if I want to trigger this popover with something manual I run into problems.  Obviously, the dom node is not an Anvil component that can have methods added to it such as ".pop()", so I am unable to trigger my popover?

Perhaps there is a way to make a dom node an Anvil component that I'm not aware of?  

Any help would be great!

Thanks

**Comment by s-cork** [comment, marked as answer]
pop is the same as popover
you can just do `pop(dom_node, "show")`

```python
from anvil_extras.popover import pop, popover
```

the tricky part is how do you trigger it?
Have you tried setting the trigger to be "click"

**Reply by chad63e** [reply]
Thanks, I had to add "pop" to the import declaration, and everything works!
